enhancements/network/user-defined-network-segmentation.md

```diff
@@ -391,6 +488,98 @@ guarantee uniqueness and eliminates the ability to falsify access.
 After creating the CRD, check the status to ensure OVN-Kubernetes has accepted this network to serve the namespaces
 selected. Now tenants may go ahead and be provisioned their namespace.
 
+#### NetworkAttachmentDefinition rendering
+
+The NAD should be created with owner-reference to the CRD, using the owner-reference mechanism should prevent deletion of the NAD before the corresponding CRD instance is deleted.
```

Comment:
here you mean controller will be created the NAD with an owner reference right? could you please clarify the bit that controller will watch for CRs and then create the corresponding NADs in each namespace?